readiness-tool/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "chrome-ready-ness-tool",
+  "version": "10.3.1",
+  "description": "A tool for the sales team to better market AMP",
```

Comment:
😄 This may be true, but it'd be better to describe what the tools does rather than how it's used.